src/pages/ticketConfirmation.tsx

```diff
@@ -210,11 +210,6 @@ export const buildTicketConfirmationElements = (
                 content: fareTypeProps.service.split('#')[0],
                 href: 'service',
             },
-            {
```

Comment:
Deleting this since it currently just shows `{ID}#{ID}` which isn't meaningful

Reply:
We need to remember to add this back in